### ISSUE TITLE
Missing translations for Drawer

### ISSUE THREAD

**Issue by petchen123** [LIT-EIA/adapt_authoring#1]
![image](https://github.com/MeD-DMC/adapt_authoring_college/assets/25251968/8cd461cb-b02b-4634-a8f9-34d278847b82)
